.werft/build.ts

```diff
@@ -405,6 +398,17 @@ export async function deployToDev(deploymentConfig: DeploymentConfig, workspaceF
         exec(`werft log result -d "dev installation" -c github url ${url}/workspaces/`);
     }
 
+    werft.log(`observability`, "Installing monitoring-satellite...")
+    if (deploymentConfig.withObservability) {
+        await installMonitoring();
+        exec(`werft log result -d "Monitoring Satellite - Grafana" -c github-check-Grafana url https://grafana-${monitoringDomain}/dashboards`);
+        exec(`werft log result -d "Monitoring Satellite - Prometheus" -c github-check-Prometheus url https://prometheus-${monitoringDomain}/graph`);
+    } else {
+        exec(`echo '"with-observability" annotation not set, skipping...'`, {slice: `observability`})
+        exec(`echo 'To deploy monitoring-satellite, please add "/werft with-observability" to your PR description.'`, {slice: `observability`})
```

Comment:
Does this comment still make sense?

Reply:
I think so. It's a nice way to advertise this feature

Reply:
What I meant is that it is not in the PR description anymore, but at the github comment 🤔